jax/experimental/array.py

```diff
@@ -401,3 +402,5 @@ def _array_local_result_handler(aval, sharding, indices):
   return lambda bufs: Array(aval, sharding, bufs, committed=True)
 pxla.local_result_handlers[(core.ShapedArray, pxla.OutputType.Array)] = _array_local_result_handler
 pxla.local_result_handlers[(core.ConcreteArray, pxla.OutputType.Array)] = _array_local_result_handler
+
+ndarray.register(Array)
```

Comment:
Small nit: Can you move this above to line 358 please?

Reply:
Done